cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapperWrapper.java

```diff
@@ -76,16 +76,14 @@ public void run(Context context) throws IOException, InterruptedException {
     MapReduceClassLoader classLoader = MapReduceClassLoader.getFromConfiguration(context.getConfiguration());
     BasicMapReduceTaskContext basicMapReduceContext = classLoader.getTaskContextProvider().get(context);
 
-    InputSplit inputSplit = context.getInputSplit();
-    if (inputSplit instanceof TaggedInputSplit) {
-      TaggedInputSplit taggedInputSplit = (TaggedInputSplit) inputSplit;
-      basicMapReduceContext.setInputName(taggedInputSplit.getName());
-    }
-
     // this is a hook for periodic flushing of changes buffered by datasets (to avoid OOME)
     WrappedMapper.Context flushingContext = createAutoFlushingContext(context, basicMapReduceContext);
 
     basicMapReduceContext.setHadoopContext(flushingContext);
+    InputSplit inputSplit = context.getInputSplit();
```

Comment:
Oh, there is no issue here. I was just reverting back to how the code was before since we moved the authorization from this class to StreamInputFormat.